cypress/support/constants.js

```diff
@@ -0,0 +1,54 @@
+import { uid } from 'uid';
+
+const constants = {
+  // learn portal base url
+  LEARN_PORTAL_BASE_URL : `${Cypress.config("learnPortalBaseUrl")}`,
+  // authoring base url
+  AUTHORING_BASE_URL : `${Cypress.config("laraBaseUrl")}`,
+  // admin user details
+  ADMIN_USERNAME : `${Cypress.config("admin")["username"]}`,
```

Comment:
Since this is following a very clear pattern, perhaps we can find a way to simplify this so new configuration options don't need to be duplicated as constants.  With some work we could setup an object that that would let you replace `ADMIN_USERNAME` with `config.admin.username`.  And then there wouldn't be a need for most of these constants.

Even if you do stick with this explicit approach you could clean it up a little by replacing:
```
`${Cypress.config("admin")["username"]}`
```
could be replaced with
```
Cypress.config("admin").username
```

Reply:
I have modified constants.js such that it now does this :

`const config = Cypress.config();
  // learn portal base url
  LEARN_PORTAL_BASE_URL : config.learnPortalBaseUrl,
  // admin user details
  ADMIN_USERNAME : config.admin.username`

I'd still like to keep them in constants.js since I feel c.ADMIN_USERNAME is a little more concise than config.admin.username